src/substrait/textplan/tests/ParseResultMatchers.h

```diff
@@ -17,6 +17,9 @@ namespace io::substrait::textplan {
 [[maybe_unused]] ::testing::Matcher<const ParseResult&> SerializesTo(
     std::string expected_result);
 
+[[maybe_unused]] ::testing::Matcher<const ParseResult&> SerializeContains(
```

Comment:
Totally a nerd snipe (feel free to ignore) but I wonder if the two `Serialize` matches could be combined to...

```
Serialized(::testing::Eq(exact_text))
Serialized(::testing::HasSubstr(contains_text))
```

Reply:
It doesn't work out of the box but I do suspect that such a thing is possible.  For now as I don't use SerializeContains in this PR I've removed it and I'll see what I can do for the PR that actually uses it.

Reply:
Done.